### ISSUE TITLE
Fujitsu SnapScan resolution not detected correctly

### ISSUE THREAD

**Issue by beat** [openpaperwork/paperwork#230]
getting following errors and no resolution displayed for the SnapScan scaner:

```
paperwork 

** (paperwork:2568): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-BEKIFYOiIi: Connection refused
INFO   paperwork.paperwork            Looking for locales in 'locale/fr/LC_MESSAGES/paperwork.mo' ...
INFO   paperwork.paperwork            Looking for locales in '/usr/local/share/locale/fr/LC_MESSAGES/paperwork.mo' ...
INFO   paperwork.paperwork            Looking for locales in '/usr/share/locale/fr/LC_MESSAGES/paperwork.mo' ...
INFO   root                           Will use locales from '/usr/share/locale'
INFO   paperwork.backend.config       Config file found: /home/beat/.config/paperwork.conf
INFO   root                           UI file used: /usr/share/paperwork/mainwindow.glade

(paperwork:2568): Gtk-WARNING **: Could not load image 'paperwork.svg': Failed to open file '/usr/share/paperwork/paperwork.svg': No such file or directory
INFO   paperwork.frontend.mainwindow  Image view port resized. (0, 0) --> (579, 705)
INFO   paperwork.frontend.jobs        [Scheduler Progress] Starting
INFO   paperwork.frontend.jobs        [Scheduler Main] Starting
INFO   paperwork.frontend.jobs        [Scheduler Progress] Started
INFO   paperwork.frontend.actions     Action: [Refresh index]
INFO   paperwork.frontend.jobs        [Scheduler Main] Started
INFO   paperwork.backend.docsearch    Opening index dir '/home/beat/.local/share/paperwork/index' ...
INFO   paperwork.frontend.mainwindow  Document examen finished. Updating index ...
INFO   paperwork.frontend.mainwindow  New document: 0
INFO   paperwork.frontend.mainwindow  Updated document: 0
INFO   paperwork.frontend.mainwindow  Deleted document: 0
INFO   paperwork.frontend.mainwindow  No changes
INFO   paperwork.frontend.mainwindow  List 'documents' : 1 elements displayed (1 additionnal)
INFO   paperwork.frontend.actions     Action: [Scan a single page]
INFO   paperwork.frontend.actions     Action: [Open settings dialog]
INFO   root                           UI file used: /usr/share/paperwork/settingswindow.glade
ERROR  paperwork.frontend.settingswindow Warning: Long name not found for language 'equ'.
WARNING paperwork.frontend.settingswindow   Will use short name as long name.
ERROR  paperwork.frontend.settingswindow Warning: Long name not found for language 'osd'.
WARNING paperwork.frontend.settingswindow   Will use short name as long name.

(paperwork:2568): Gtk-CRITICAL **: gtk_list_store_get_value: assertion `column < priv->n_columns' failed
/usr/lib/python2.7/dist-packages/gi/types.py:113: Warning: g_object_set_property: assertion `G_IS_VALUE (value)' failed
  return info.invoke(*args, **kwargs)
/usr/lib/python2.7/dist-packages/gi/types.py:113: Warning: g_value_unset: assertion `G_IS_VALUE (value)' failed
  return info.invoke(*args, **kwargs)

(paperwork:2568): Gtk-CRITICAL **: gtk_list_store_get_value: assertion `column < priv->n_columns' failed

(paperwork:2568): Gtk-CRITICAL **: gtk_list_store_get_value: assertion `column < priv->n_columns' failed

(paperwork:2568): Gtk-CRITICAL **: gtk_list_store_get_value: assertion `column < priv->n_columns' failed
INFO   paperwork.frontend.jobs        [Scheduler progress] Starting
INFO   paperwork.frontend.jobs        [Scheduler progress] Started
INFO   paperwork.frontend.settingswindow Looking for scan devices ...

(paperwork:2568): Gtk-CRITICAL **: gtk_list_store_get_value: assertion `column < priv->n_columns' failed

(paperwork:2568): Gtk-CRITICAL **: gtk_list_store_get_value: assertion `column < priv->n_columns' failed

(paperwork:2568): Gtk-CRITICAL **: gtk_list_store_get_value: assertion `column < priv->n_columns' failed

(paperwork:2568): Gtk-CRITICAL **: gtk_list_store_get_value: assertion `column < priv->n_columns' failed

(paperwork:2568): Gtk-CRITICAL **: gtk_list_store_get_value: assertion `column < priv->n_columns' failed

(paperwork:2568): Gtk-CRITICAL **: gtk_list_store_get_value: assertion `column < priv->n_columns' failed
INFO   paperwork.frontend.settingswindow Device found: [FUJITSU ScanSnap S1500] -> [fujitsu:ScanSnap S1500:9226]
INFO   paperwork.frontend.settingswindow Device found: [Epson RX420] -> [epson2:libusb:001:047]
WARNING paperwork.frontend.jobs        Job DeviceFinder:0 took 2629ms and is unstoppable ! (maximum allowed: 500ms)
INFO   paperwork.frontend.settingswindow Got value [['FUJITSU ScanSnap S1500', 'fujitsu:ScanSnap S1500:9226']]
INFO   paperwork.frontend.settingswindow Got value [['Epson RX420', 'epson2:libusb:001:047']]
INFO   paperwork.frontend.settingswindow Select scanner: 0
/usr/lib/python2.7/dist-packages/gi/types.py:113: Warning: unable to set property `xpad' of type `guint' from value of type `gchararray'
  return info.invoke(*args, **kwargs)
INFO   paperwork.frontend.settingswindow Looking for resolution of device [fujitsu:ScanSnap S1500:9226]
ERROR  paperwork.frontend.jobs        ===> Job ResolutionFinder:0 raised an exception: <type 'exceptions.TypeError'>: not all arguments converted during string formatting
ERROR  paperwork.frontend.jobs         0: /usr/local/lib/python2.7/dist-packages/paperwork/frontend/jobs.py: L  158: _run
ERROR  paperwork.frontend.jobs         1: /usr/local/lib/python2.7/dist-packages/paperwork/frontend/settingswindow.py: L  163: do
ERROR  paperwork.frontend.jobs        ---> Job ResolutionFinder:0 was started by:
ERROR  paperwork.frontend.jobs         0: /usr/local/bin/paperwork: L    6: <module>
ERROR  paperwork.frontend.jobs         1: /usr/local/lib/python2.7/dist-packages/paperwork/paperwork.py: L   98: main
ERROR  paperwork.frontend.jobs         2: /usr/lib/python2.7/dist-packages/gi/types.py: L  113: function
ERROR  paperwork.frontend.jobs         3: /usr/local/lib/python2.7/dist-packages/paperwork/frontend/settingswindow.py: L  662: on_finding_end_cb
ERROR  paperwork.frontend.jobs         4: /usr/lib/python2.7/dist-packages/gi/types.py: L  113: function
ERROR  paperwork.frontend.jobs         5: /usr/local/lib/python2.7/dist-packages/paperwork/frontend/actions.py: L   74: on_combobox_changed_cb
ERROR  paperwork.frontend.jobs         6: /usr/local/lib/python2.7/dist-packages/paperwork/frontend/actions.py: L   53: __do
ERROR  paperwork.frontend.jobs         7: /usr/local/lib/python2.7/dist-packages/paperwork/frontend/settingswindow.py: L  366: do
ERROR  paperwork.frontend.jobs         8: /usr/local/lib/python2.7/dist-packages/paperwork/frontend/jobs.py: L  227: schedule
INFO   paperwork.backend.config       Writing /home/beat/.config/paperwork.conf ... 
INFO   paperwork.backend.config       Done
INFO   paperwork.frontend.settingswindow Settings window destroyed
INFO   paperwork.frontend.jobs        [Scheduler progress] Stopping
INFO   paperwork.frontend.jobs        [Scheduler progress] Stopped
INFO   paperwork.frontend.actions     Action: [Scan multiples pages]
INFO   root                           UI file used: /usr/share/paperwork/multiscan.glade
INFO   paperwork.frontend.actions     Action: [Add doc to the multi-scan list]
INFO   paperwork.frontend.actions     Action: [Doc selected in multi-scan list]
INFO   paperwork.frontend.actions     Action: [Start multi-scan]
INFO   paperwork.backend.config       Scanner mode set to 'Color'
INFO   paperwork.util                 Setting scanner option 'source' to 'ADF Front'
INFO   paperwork.backend.img.page     Scanner resolution: 300
INFO   paperwork.backend.img.page     Scanner calibration: None
INFO   paperwork.backend.img.page     Scan rotated of 0 degree
INFO   paperwork.backend.img.page     Scan rotated of -90 degree
INFO   paperwork.backend.img.page     Scan rotated of -180 degree
INFO   paperwork.backend.img.page     Scan rotated of -270 degree
INFO   paperwork.backend.img.page     Using Tesseract for OCR
INFO   paperwork.backend.img.page     Running OCR on page orientation 0
INFO   paperwork.backend.img.page     Running OCR on page orientation 1
INFO   paperwork.backend.img.page     Evaluating score of page orientation (0) using method 'spell_checker' ...
INFO   paperwork.backend.img.page     Evaluating score of page orientation (1) using method 'spell_checker' ...
INFO   paperwork.backend.img.page     Page orientation 0 score: 0
INFO   paperwork.backend.img.page     Page orientation 1 score: 0
INFO   paperwork.backend.img.page     Running OCR on page orientation 2
INFO   paperwork.backend.img.page     Running OCR on page orientation 3
INFO   paperwork.backend.img.page     Evaluating score of page orientation (3) using method 'spell_checker' ...
INFO   paperwork.backend.img.page     Evaluating score of page orientation (2) using method 'spell_checker' ...
INFO   paperwork.backend.img.page     Page orientation 3 score: 0
INFO   paperwork.backend.img.page     Page orientation 2 score: 0
INFO   paperwork.backend.img.page     Best: 0.000000
INFO   paperwork.backend.img.page     Extracting boxes ...
INFO   paperwork.backend.img.page     Done
INFO   paperwork.backend.img.page     Scan done
INFO   paperwork.backend.docsearch    Updating modified doc: 20130816_1755_40
WARNING paperwork.backend.img.page     WARNING: Doc 20130816_1755_40 uses old box format
INFO   paperwork.backend.docsearch    Index: Commiting changes
INFO   paperwork.backend.docsearch    Adding document '20130816_1755_40' to the index
INFO   paperwork.frontend.mainwindow  List 'pages' : 1 elements displayed (1 additionnal)
INFO   root                           Showing page 20130816_1755_40 p1
INFO   paperwork.frontend.actions     Action: [Open selected document]
INFO   paperwork.frontend.mainwindow  No document selected. Can't open
INFO   paperwork.frontend.mainwindow  List 'documents' : 2 elements displayed (2 additionnal)
INFO   paperwork.frontend.mainwindow  Zoom: 0.175455
WARNING paperwork.backend.img.page     WARNING: Doc 20130816_1755_40 uses old box format
INFO   paperwork.frontend.multiscan   Multi-scan dialog destroyed
INFO   paperwork.frontend.actions     Action: [Doc selected in multi-scan list]
WARNING paperwork.frontend.multiscan   No doc selected
```
